mob/mob.c

```diff
@@ -16,6 +19,8 @@ mobile* make_mob(struct Level *lvl) {
     mob->lvl = lvl;
     mob->x = 0;
     mob->y = 0;
+    //TODO Is this really a magic number? I think it's maybe
```

Comment:
Nothing magic about it. Presumably it will always get reset to something else but this is the lowest value where the mob won't die on the first turn. Seems like a sensible default.